templates/layout.html

```diff
@@ -9,6 +9,7 @@
 
 <link rel="manifest" href={{ url_for('static', filename='webmanifest.json') }}>
 
+<link rel=stylesheet href=https://www.gitcdn.xyz/repo/neomad-team/styleguide/master/content/styles/main.css>
```

Comment:
not ready yet ... are you not worried about conflict with neomad.org css ? I made a lot of changements